scripts/Muon/GUI/Common/corrections_tab_widget/dead_time_corrections_view.py

```diff
@@ -0,0 +1,117 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+from mantidqt.utils.qt import load_ui
+from Muon.GUI.Common.utilities.muon_file_utils import filter_for_extensions, show_file_browser_and_return_selection
+
+from qtpy.QtWidgets import QWidget
+
+ui_form, widget = load_ui(__file__, "dead_time_corrections_view.ui")
+
+DEAD_TIME_DATA_FILE = "From data file"
```

Comment:
should we import these from the model?

Reply:
These are not used by the model and only for the view so I think they are better kept in this file